ClientTests/SearchEnginesTests.swift

```diff
@@ -28,7 +28,7 @@ class SearchEnginesTests: XCTestCase {
         // If this is our first run, Google should be first for the en locale.
         let profile = MockProfile()
         let engines = SearchEngines(prefs: profile.prefs, files: profile.files)
-        XCTAssertEqual(engines.defaultEngine.shortName, DefaultSearchEngineName)
+        XCTAssertEqual(engines.defaultEngine().shortName, DefaultSearchEngineName)
```

Comment:
Should we add unit tests for private search engines too?

Reply:
Good idea, I added one test and one more assertion to verify that default private engine is set.